.github/workflows/ci-scheduling.yml

```
@@ -0,0 +1,128 @@
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      
+      - name: Node.js セットアップ
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+      
+      - name: キャッシュ復元
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)